pkgs/racket-doc/scribblings/inside/custodians.scrbl

```diff
@@ -51,16 +51,18 @@ be remembered until either the custodian shuts it down or
 zero, the value is allowed to be garbage collected (and automatically
 removed from the custodian).
 
-Independent of whether @var{strong} is zero, the value @var{o} is
-initially weakly held. A value associated with a custodian can
-therefore be finalized via will executors.
+Independent of whether @var{strong} is zero, the value @var{o}
+is initially weakly held, but becomes strongly held when
```

Comment:
Because you changed `if` to `when`.

I'm thinking, "the lifetime of `o` is predictable, initially it's weakly held and once the GC finds it it's strongly held"